tests/puppeteer/campaigns/sanity/02_productsBO/01_filterProducts.js

```diff
@@ -60,8 +65,9 @@ describe('Filter in Products Page', async () => {
     {args: {filterBy: 'reference', filterValue: Products.demo_1.reference}},
     {args: {filterBy: 'category', filterValue: Categories.men.name}},
   ];
-  tests.forEach((test) => {
+  tests.forEach((test, index) => {
     it(`should filter list by ${test.args.filterBy} and check result`, async function () {
+      await testContext.addContextItem(this, 'stepIdentifier', `filterBy${index + 1}`, baseContext);
```

Comment:
should it be 
```suggestion
      await testContext.addContextItem(this, 'stepIdentifier', `filterBy${test.args.filterBy}`, baseContext);
```

Reply:
`test.args.filterBy` is something that could change, but the hole idea is to not change the step Identifier.
Example :  `test.args.filterBy = name` and used in selectors as `input#name`, if selector change to `name1` than step Identifier will change to.

Reply:
Good explication :heart:

Reply:
Another solution might be to use a parameter in the `tests` object, used only for this context purpose. WDYT ?

Reply:
it would be nice yeah to have something like 
```js
 const tests = [
    {args: {identifier: 'filterName', filterBy: 'name', filterValue: Products.demo_14.name}},
```

Reply:
done.
I didn't add identifier in `05_deleteProductsWithBulkActions` (no need to), using index is better.